.editorconfig

```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Please remove this comment.

Reply:
I removed it as requested

It's worth noting that [it's the default header that appear in their docs](https://editorconfig.org) and shows support to the project, which is nice IMO...